Doc/library/calendar.rst

```diff
@@ -170,6 +170,64 @@ it's the base calendar for all computations.
       sheet should be used. *encoding* specifies the encoding to be used for the
       output (defaulting to the system default encoding).
 
+   :class:`HTMLCalendar` has the following attributes you can override to
+   customize the style of the calendar:
+
+   .. attribute:: cssclasses
+
+      A list of CSS styles used for each weekday. The default style list is::
+
+         cssclasses = ["mon", "tue", "wed", "thu", "fri", "sat", "sun"]
+
+      more styles can be added for each day::
+
+         cssclasses = ["mon text-bold", "tue", "wed", "thu", "fri", "sat", "sun red"]
+
+      Note that the length of this list must be seven items.
+
+   .. attribute:: noday_classes
+
+      The CSS class for a weekday occurring in the previous or coming month.
+
+   .. attribute:: weekday_head_classes
+
+      A list of CSS styles used for each weekday. The default style list is
+      the same as :attr:`cssclasses`.
+
+   .. attribute:: month_head_classes
+
+      A space separated list of styles for the month head, for example::
+
+      "text-bold text-red"
+
+      The default value is ``"month"``.
+
+   .. attribute:: month_classes
+
+      A space separated list of styles for the whole table.
+      The default value is ``"month"``.
+
+   .. attribute:: year_classes
+
+      A space separated list of styles for the table when formatting the year
```

Comment:
-> "A space separated list of CSS class names used for the year label above the year table" or something like that.